<launch>
	<node pkg="tf" type="static_transform_publisher" name="map_to_odom_robot_0" args="-7.593280 -6.066174 0.000000 -140.902763 0.0 0.0 map robot_0/odom 20"/>
	<node pkg="tf" type="static_transform_publisher" name="map_to_odom_robot_1" args="10.616959 -11.189331 0.000000 68.669580 0.0 0.0 map robot_1/odom 20"/>
	<node pkg="tf" type="static_transform_publisher" name="map_to_odom_robot_2" args="-4.173999 -8.408521 0.000000 -130.887251 0.0 0.0 map robot_2/odom 20"/>
	<node pkg="tf" type="static_transform_publisher" name="map_to_odom_robot_3" args="5.275074 5.158360 0.000000 81.298016 0.0 0.0 map robot_3/odom 20"/>
	<node pkg="tf" type="static_transform_publisher" name="map_to_odom_robot_4" args="10.561646 -1.711718 0.000000 72.178056 0.0 0.0 map robot_4/odom 20"/>
</launch>